lib/open_project/help_link/engine.rb

```diff
@@ -26,8 +26,12 @@ class Engine < ::Rails::Engine
     engine_name :openproject_help_link
 
     def self.settings
-      { :default => {"help_link_target" => "https://www.openproject.org/projects/support"},
-        :partial => "settings/openproject_help_link_settings.html.erb" }
+      {
+        default: {
+          'help_link_target' => 'https://www.openproject.org/help'
+        },
+        partial: 'settings/openproject_help_link_settings.html.erb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.